[not ready] Punchcards by @arunasank

geohacker opened the pull request.

Commit:
index.html - Comment out the nv.d3.css file.

Don't need it since the new code doesn't depend on nv.d3

Commit:
style.css -> Change the height to 500px

Commit:
app.js -> Add comments - minor

Commit:
app.js -> New d3 implementation for the graph

Commit:
app.js -> Show warning if the time interval selected > a week

geohacker committed:
use manual tickValues

geohacker committed:
remove all the date nonsense and make peace with timezones

geohacker commented:
@arunasank I'm seeing more issues -
1. Axis needs to change based on time range.
2. When you read along Y axis, should be able to see who did better in particular point in time.

arunasank commented:
Also:

Circle scales need to vary in the overall graph and not just on a particular x axis.

Commit:
app.js -> Make data appear when user hovers over an axis tick

This helps the user see all the data for a given date/hour/month etc as opposed
to per user

Commit:
app.js -> Fix issue that came up because of 0e6c085b535b9f525186905f

Hovering on names of data team people stopped making the circle text from
appearing. Fix this by adding both classes for the text at once.

Commit:
Code changes so osm-edit-report installs

Commit:
refactor d3 functions and rewrite lots of code

Commit:
Update height to 600px

Commit:
Comment out window.event line of the code
